treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl

```diff
@@ -302,12 +302,11 @@ DROP TABLE IF EXISTS `series_signature`;
 CREATE TABLE `series_signature` (
   `signature` varchar(50) COLLATE utf8_bin NOT NULL,
   `property` varchar(50) COLLATE utf8_bin NOT NULL,
-  `value` varchar(150) COLLATE utf8_bin NOT NULL,
+  `value` text COLLATE utf8_bin NOT NULL,
   `active_status` enum('active','onhold','deleted') COLLATE utf8_bin DEFAULT 'active',
   KEY `idx_signature` (`signature`),
   KEY `idx_property` (`property`),
-  KEY `idx_value` (`value`),
```

Comment:
I think we agreed after chatting on irc that we should leave this index out. We can always put the index back later if we really want it. Properly indexing performance series is a complicated problem and I don't have the full solution to it yet, but it surely is much more than just putting an index on a column in this table. :)